File: internal/content/testdata/bench/csharp/max-gap-collectible-items.cs
public class Solution {
    public long collectible_count(long[] rain_times, long[] dry_durations) {
        long maxGap = 0;
        for (int i = 0; i < rain_times.Length - 1; i++) {
            long gap = rain_times[i + 1] - rain_times[i];
            if (gap > maxGap) maxGap = gap;
        }
        long count = 0;
        foreach (long d in dry_durations) {
            if (d <= maxGap) count++;
        }
        return count;
    }
}

File: internal/content/testdata/bench/go/max-gap-collectible-items.go
func collectible_count(rain_times []int, dry_durations []int) int {
	maxGap := 0
	for i := 0; i < len(rain_times)-1; i++ {
		gap := rain_times[i+1] - rain_times[i]
		if gap > maxGap {
			maxGap = gap
		}
	}
	count := 0
	for _, d := range dry_durations {
		if d <= maxGap {
			count++
		}
	}
	return count
}
